web/Views/Admin/_MatchListTable.cshtml

```diff
@@ -0,0 +1,82 @@
+﻿@model TenantMatchListModel
+
+@functions
+{
+    static string GetPreferenceFont(bool? preference)
+    {
+        return preference == true ? "" : "font-orange";
+    }
+
+    static string GetPreferenceIcon(bool? preference)
+    {
+        return preference == true ? "bi-check-lg" : "bi-x-lg";
+    }
+}
+
+@if (Model.TenantList!.Count == 0)
+{
+    <h2 class="font-h3">There are no tenants which match these search criteria.</h2>
+    return;
```

Comment:
No return needed either :)

Reply:
I think we want the return here so that if there are no tenants it doesn't display the table headers.

Reply:
Sure